music21/analysis/discrete.py

```diff
@@ -1348,6 +1299,11 @@ def analyzeStream(streamObj, *args, **keywords):
     if args:
         method = args[0]
 
+    if method == 'range':
+        # getPitchRanges() was removed in v7
+        # this synonym is being added for compatibility
+        method = 'span'
+
```

Comment:
I guess this is not 100% great in that someone who extended the functionality with another analysis method would have to override analyzeStream() also in order to not get this synonym.

Reply:
Ohhhh but if I get rid of the synonym then I'm signing up for rewriting the user's guide examples. I'll give Myke time to weigh in.

Reply:
> I guess this is not 100% great in that someone who extended the functionality with another analysis method would have to override analyzeStream() also in order to not get this synonym.

On second thought, this isn't so bad. The name was previously reserved because m21 was using it. And someone can override this OR override analysisClassFromMethodName. I think it's okay. Better to support backwards compatibility for now.